azure-pipelines.yml

```diff
@@ -9,188 +9,89 @@ trigger:
-              value: '.NET Performance - CoreClr $(_RunType) - $(Build.SourceBranchName) $(Build.SourceVersion)'
-            - name: UploadToBenchview
-              value: '--upload-to-benchview-container coreclr'
-            - group: DotNet-HelixApi-Access
-            - group: dotnet-benchview
-          workspace:
-            clean: all
-          pool:
-            name: Hosted Ubuntu 1604
-          container: ubuntu_x64_build_container
-          strategy:
-            matrix:
-              x64_CoreCLR_netcoreapp30:
-                _Architecture: x64
-                _Category: CoreCLR
-                _Framework: netcoreapp3.0
-                _BuildConfig: $(_Architecture)_$(_Category)_$(_Framework)
-                _BenchViewArguments: '--benchview-submission-name "$(BenchviewCommitName)" --benchview-machinepool perfsnake --generate-benchview-data --benchview-run-type $(_RunType) $(UploadToBenchview)'
-              x64_CoreCLR_netcoreapp22:
-                _Architecture: x64
-                _Category: CoreCLR
-                _Framework: netcoreapp2.2
-                _BuildConfig: $(_Architecture)_$(_Category)_$(_Framework)
-                _BenchViewArguments: ''
-              x64_CoreCLR_netcoreapp21:
-                _Architecture: x64
-                _Category: CoreCLR
-                _Framework: netcoreapp2.1
-                _BuildConfig: $(_Architecture)_$(_Category)_$(_Framework)
-                _BenchViewArguments: ''
-              x64_CoreCLR_netcoreapp20:
-                _Architecture: x64
-                _Category: CoreCLR
-                _Framework: netcoreapp2.0
-                _BuildConfig: $(_Architecture)_$(_Category)_$(_Framework)
-                _BenchViewArguments: ''
-          steps:
-          - checkout: self
-            clean: true
-          - template: /eng/common/templates/steps/send-to-helix.yml
-            parameters:
-              HelixSource: 'pr/dotnet/performance/$(Build.SourceBranch)' # sources must start with pr/, official/, prodcon/, or agent/
-              HelixType: 'test/performance_$(_BuildConfig)/'
-              HelixAccessToken: $(HelixApiAccessToken)
-              HelixTargetQueues: Ubuntu.1604.Amd64.Perf
-              WorkItemDirectory: $(Build.SourcesDirectory)/src/benchmarks/micro
-              WorkItemCommand: python3.5 $HELIX_CORRELATION_PAYLOAD/benchmarks_ci.py $(_BenchViewArguments) --working-directory $HELIX_WORKITEM_PAYLOAD --bin-directory $HELIX_WORKITEM_PAYLOAD --incremental no --category $(_Category) --architecture $(_Architecture) -f $(_Framework) --bdn-arguments="--buildTimeout 300"
-              WorkItemTimeout: 14400 # 4 hours
-              CorrelationPayloadDirectory: $(Build.SourcesDirectory)/scripts
-              EnableXUnitReporter: false
+
+# Windows x64 micro benchmarks, public correctness job
+- ${{ if eq(variables['System.TeamProject'], 'public') }}:
+  - template: /eng/performance/benchmark_jobs.yml
+    parameters:
+      name: windows
```

Comment:
What is the version for this and line 34?

Reply:
I added the versions and made it mandatory to always provide them

Reply:
Thank you!